tree/ntuple/v7/test/ntuple_show.cxx

```diff
@@ -120,8 +120,12 @@ TEST(RNTupleShow, BasicTypes)
    // clang-format on
    EXPECT_EQ(fString1, os1.str());
 
-   // TODO(jblomer): this should fail to an exception instead
-   EXPECT_DEATH(ntuple2->Show(2), ".*");
+   try {
+      ntuple2->LoadEntry(2);
+      FAIL() << "loading a non-existing entry should throw";
+   } catch (const RException &err) {
+      EXPECT_THAT(err.what(), testing::HasSubstr("entry with index 2 out of bounds"));
+   }
```

Comment:
Should we use `EXPECT_THROW`? It wouldn't allow checking the error message though...

Reply:
Both are used throughout the tests. In the other two instances in this PR (`ntuple_storage` and `ntuple_storage_daos`) I would prefer to keep it like this because it is used that way in the other test cases in those files as well, but for `ntuple_show` I'm fine to change it to `EXPECT_THROW`, the error message will then be checked by those other two test cases after all.

Reply:
Ok, `grep` found `EXPECT_THROW` in `ntuple_storage_daos.cxx` for one call to `LoadEntry`, but you are right that the rest of `ntuple_storage.cxx` and especially `ntuple_basics.cxx` don't use it. Fine with me to leave it like this everywhere.